xl-v1-utils.go

```diff
@@ -78,14 +78,14 @@ func reduceErrs(errs []error) error {
 
 // Validates if we have quorum based on the errors with errDiskNotFound.
 func isQuorum(errs []error, minQuorumCount int) bool {
-	var diskFoundCount int
+	var errCount int
 	for _, err := range errs {
-		if err == errDiskNotFound {
+		if err == errDiskNotFound || err == errFaultyDisk || err == errDiskAccessDenied {
 			continue
 		}
-		diskFoundCount++
+		errCount++
 	}
-	return diskFoundCount >= minQuorumCount
+	return errCount >= minQuorumCount
```

Comment:
instead of errCount, it can be named just count.
Because it reads as `if error count is more than quorumCount we have met the quorum` when it should mean the opposite